src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class NumberSupplier {
+    public int getRandomNumber() {
+        return (int) ((Math.random() * 100));
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/198043335-ee713f37-a60e-4e25-84d1-cc36939c0441.png)

100 is a magic number